scripts/joinmarket-qt.py

```diff
@@ -1556,9 +1556,8 @@ def updateWalletInfo(self, walletinfo=None):
             # if expansion states existed, reinstate them:
             if len(previous_expand_states) == max_mixdepth_count:
                 m_item.setExpanded(previous_expand_states[mixdepth][0])
-            # by default, if the mixdepth is 0 or balance of the mixdepth is
-            # greater than 0, expand it
-            elif mixdepth == 0 or float(mdbalance) > 0:
```

Comment:
The conversion of `float(mdbalance)` could fail now if the mdbalance is of format `x(y)`.
Since we're still collapsed at the "external / internal" level, it's fine to expand at the mixdepth level by default.

Reply:
Yes that's fine. The additional expansion just shows the xpub so it's not exactly overwhelming.